encoding/Cargo.toml

```diff
@@ -13,7 +13,7 @@ fvm_shared         = { version = "0.8.0", default-features = false }
 fvm_ipld_encoding  = "0.2.1"
 blake2b_simd       = "1.0"
 serde              = { version = "1.0", features = ["derive"] }
-serde_bytes        = { package = "cs_serde_bytes", version = "0.12" }
+cs_serde_bytes     = "0.12"
```

Comment:
In general, why do we need this custom `serde_bytes` crate? What are the differences? Shall we make a task to take a look a around? I looked at [crates.io](https://crates.io/crates/cs_serde_bytes) and seems it doesn't even point to a correct repository.

Reply:
That is a very good question. It seems that we're maintaining a fork of `serde_bytes` here: https://github.com/ChainSafe/serde-bytes. This crate is also used by the FVM. We should create an issue to identify why this crate exists and if it can be removed (from both Forest and ref-fvm).

Reply:
Issue created: #1722